### DISCUSSION TITLE
New installation not running

### DISCUSSION THREAD

**Discussion by JeffreyRoberson** [akvorado/akvorado#214] [Q&A]
Docker instances are not running or staying up.

I built a fresh Debian 11 system, installed Docker from their website to ensure I had the latest versions.  However, not all of the services are working or will stay up.  Here is what docker-compose ps shows:

akvorado_akvorado-conntrack-fixer_1   /usr/local/bin/akvorado co ...   Up                                                                       
akvorado_akvorado-console_1           /usr/local/bin/akvorado co ...   Restarting                                                               
akvorado_akvorado-inlet_1             /usr/local/bin/akvorado in ...   Restarting                                                               
akvorado_akvorado-orchestrator_1      /usr/local/bin/akvorado or ...   Restarting                                                               
akvorado_akvorado-service_1           /usr/local/bin/akvorado ve ...   Exit 0                                                                   
akvorado_clickhouse_1                 /entrypoint.sh                   Up (unhealthy)   8123/tcp, 9000/tcp, 9009/tcp                            
akvorado_geoip_1                      /usr/bin/entry.sh                Up                                                                       
akvorado_kafka-ui_1                   /bin/sh -c java $JAVA_OPTS ...   Up               8080/tcp                                                
akvorado_kafka_1                      /opt/bitnami/scripts/kafka ...   Up (healthy)     9092/tcp                                                
akvorado_traefik_1                    /entrypoint.sh --api=true  ...   Up               80/tcp, 127.0.0.1:8080->8080/tcp, 0.0.0.0:8081->8081/tcp
akvorado_zookeeper_1                  /opt/bitnami/scripts/zooke ...   Up               2181/tcp, 2888/tcp, 3888/tcp, 8080/tcp    

Also, on the docker-compose up screen, it seem to be stuck on this:
 
clickhouse_1                | + wget --no-proxy -qO /tmp/init.sh http://akvorado-orchestrator:8080/api/v0/orchestrator/clickhouse/init.sh
clickhouse_1                | + continue
clickhouse_1                | + [ ! -s /tmp/init.sh ]
clickhouse_1                | + sleep 1
clickhouse_1                | + wget --no-proxy -qO /tmp/init.sh http://akvorado-orchestrator:8080/api/v0/orchestrator/clickhouse/init.sh
clickhouse_1                | + continue
clickhouse_1                | + [ ! -s /tmp/init.sh ]
clickhouse_1                | + sleep 1
akvorado-orchestrator_1     | Error: invalid configuration:
akvorado-orchestrator_1     | invalid key "ClickHouse.network-sources"
akvorado-orchestrator_1     | invalid key "Console[0].Database.saved-filters"
akvorado_akvorado-orchestrator_1 exited with code 1

Here are my Docker versions:
docker-compose version 1.25.0, build unknown
docker-py version: 4.1.0
CPython version: 3.9.2
OpenSSL version: OpenSSL 1.1.1n  15 Mar 2022

Client:
 Version:           20.10.5+dfsg1
 API version:       1.41
 Go version:        go1.15.15
 Git commit:        55c4c88
 Built:             Mon May 30 18:34:49 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.5+dfsg1
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.15.15
  Git commit:       363e9a8
  Built:            Mon May 30 18:34:49 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.13~ds1
  GitCommit:        1.4.13~ds1-1~deb11u2
 runc:
  Version:          1.0.0~rc93+ds1
  GitCommit:        1.0.0~rc93+ds1-5+deb11u2
 docker-init:
  Version:          0.19.0
  GitCommit:        

Any ideas on what I need to fix?  Thank you.

**Comment by JeffreyRoberson** [comment]
I downgraded from 1.6.1 to 1.6.0 and that did start up.

**Reply by vincentbernat** [reply]
It looks like more you were running the configuration from the main branch, not 1.6.1. Be sure to use the tarball (or checkout the right tag).

**Reply by JeffreyRoberson** [reply]
That was the problem.  Downloading and running from the tarball fixed the problem.  I was cloning the project originally.  Thank you!
